src/store/todo/todo.reducer.ts

```diff
@@ -0,0 +1,256 @@
+import { AnyAction } from '@reduxjs/toolkit';
+import { SortTypes, TodoDTO, TodosColumns } from '~types/todo.types';
+
+import { actions } from './todo.actions';
+
+export type TodoState = {
```

Comment:
I would also refactor this file somehow, it's very big. Let's use redux slices here. Also it would be great to combine them same as actions

Reply:
Rewrite with redux slice